Lesson4/src/service/MilitaryOffice.java

```diff
@@ -0,0 +1,29 @@
+package service;
+
+import model.Person;
+import java.util.List;
+
+public class MilitaryOffice {
+    private List<Person> people;
+    private Person person;
+    public MilitaryOffice(List<Person> people) {
```

Comment:
I think you misunderstood a task a bit. You need to add class of PersonRegistry that we created on the lesson and work with it here ( get people by the address from the registry and then figure out who's fit for the military service

Reply:
Oh, I understood that later. I have redone all the homework, please check it out.